public/assets/css/style.css

```diff
@@ -51,11 +57,17 @@ li.added-class {
   padding: 0;
   position: absolute;
   top: 50%;
+  left: 105%;
   transform: translateY(-50%);
   right: 0;
-  color: #ea2929;
+  color: pink;
```

Comment:
The pink color is too pale, almost hard to see.  Please change it to rgba(234, 41, 41, 0.6);  It's the same color as #ea2929, except with lower opacity, and I'll approve it.  Otherwise, good job!